apps/root/src/state/create-position/reducer.ts

```diff
@@ -78,6 +79,15 @@ export default createReducer(initialState, (builder) => {
     .addCase(setModeType, (state, { payload }) => {
       state.modeType = payload;
     })
+    .addCase(setDCADefaultChainId, (state, { payload }) => {
```

Comment:
What's the difference from this to setDCAChainId?

Reply:
setDCAChainID also resets the whole form. And I needed frequencyValue to be persistent. Now I'm thinking that this can be one action as it was, and persist frequency value an type even if network changes